packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -27,76 +23,79 @@ export interface BaseCardProps {
    * Class names to be appended to the className prop of the component
    */
   className?: string;
+  /**
+   * Child nodes to be rendered in the component
+   */
+  children?: ReactNode;
 }
 
-interface CardPropTypes extends BaseCardProps {
+export interface CardProps extends BaseCardProps {
   /**
-   * The action to be performed on click of the Card component
+   * The action to be performed when user clicks on the Card
    */
-  onClick?: MouseEventHandler<HTMLElement>;
+  onClick?: (e: MouseEvent<HTMLElement> | KeyboardEvent<HTMLElement>) => void;
   /**
    * Applies padding styles of different sizes
    */
   padding?: 'default' | 'large' | 'none';
   /**
-   * The title of the entry
+   * The title of the entry. It will also be used as aria-label
    */
   title?: string;
   /**
    * Any additional styles that are being applied
    */
   style?: React.CSSProperties;
-  children: React.ReactNode;
+  /**
+   * If the card is selectable and has no title, it will need a aria-label to help screen readers identify it
+   */
+  ariaLabel?: string;
 }
 
-const defaultProps: Partial<CardPropTypes> = {
-  padding: 'default',
-  testId: 'cf-ui-card',
-  selected: false,
-};
-
-export class Card extends Component<CardPropTypes> {
-  static defaultProps = defaultProps;
+export const Card = ({
+  className,
+  testId = 'cf-ui-card',
+  children,
+  href,
+  target,
+  onClick,
+  padding = 'default',
+  selected = false,
+  ariaLabel,
+  ...otherProps
+}: CardProps) => {
+  const classNames = cn(styles.Card, className, {
+    [styles[`Card--padding-${padding}`]]: padding,
+    [styles['Card--is-interactive']]: onClick || href,
+    [styles['Card--is-selected']]: selected,
+  });
 
-  handleClick = (event: ReactMouseEvent<HTMLElement>) => {
-    if (!this.props.onClick) return;
+  let Element: React.ElementType = 'div';
 
-    this.props.onClick(event);
-  };
-
-  render() {
-    const {
-      className,
-      testId,
-      children,
-      href,
-      target,
-      onClick,
-      padding,
-      selected,
-      ...otherProps
-    } = this.props;
+  if (href) {
+    Element = 'a';
+  } else if (onClick) {
+    Element = 'button';
+  }
```

Comment:
@mshaaban0 and @denkristoffer 
I implemented the changes about the keyDown event, but then I started reading the ARIA links Moe sent to me and they kinda made me think "Wait a minute... I think we should use a `<button>` when the card has onClick instead of `div`"

so I changed it to do exactly that... and because it's a button, we do not need onKeyDown or tabindex

we still need aria-label and a aria-pressed

WDYT?
P.S.: this makes the selectable card very similar to the ToggleButton, but with a11y features working
I will update ToggleButton in another PR, but maybe in Forma36 v4 we will have to think if we need both 👀

Reply:
even better 👏
> we still need aria-label and a aria-pressed
+1

Reply:
Yes absolutely, huge +1! 😄